runtime/doc/vi_diff.txt

```diff
@@ -59,8 +59,6 @@ Support for different systems.
 	Vim can be used on:
 	- All Unix systems (it works on all systems it was tested on, although
 	  the GUI and Perl interface may not work everywhere).
-	- MS-DOS in real-mode (no additional drivers required).
-	- In protected mode on Windows 3.1 and MS-DOS (DPMI driver required).
 	- Windows 95 and Windows NT, with support for long file names.
```

Comment:
LOL no. But we need to fix a ton of Windows 95/98/ME/NT references too. So I left this line.